src/scaffold/zome.rs

```diff
@@ -345,6 +345,7 @@ pub fn add_common_zome_dependencies_to_workspace_cargo(
         &format!("={}", versions::HDK_VERSION),
     )?;
     let file_tree = add_workspace_external_dependency(file_tree, "serde", "1.0")?;
+    let file_tree = add_workspace_external_dependency(file_tree, "home", "=0.5.9")?;
```

Comment:
I can't add a removal comment here, people will have to figure this one out

Reply:
Yea, this is handled programatically, maybe a top level comment in the workspace cargo toml here https://github.com/holochain/scaffolding/blob/ce3f9daee5709f103b28033e591b1bf3a242706c/src/scaffold/app/cargo.rs#L12

Reply:
Doesn't seem to work. I'm thinking the TOML load/write is not preserving comments

Reply:
Ah shucks, I think the comments in the workspace crates should be a good indicator to remove the workspace dependency when upgrading, we can leave this as is for now